modules/cli/src/main/scala/scala/cli/commands/RestrictedCommandsParser.scala

```diff
@@ -57,6 +57,16 @@ object RestrictedCommandsParser {
               if arg.isExperimental && !shouldSuppressExperimentalWarnings =>
             logger.experimentalWarning(passedOption, FeatureType.Option)
             r
+          case (r @ Right(Some(_, arg: Arg, _)), passedOption :: _)
+              if arg.isDeprecated =>
+            // TODO implement proper option/option alias deprecation logic, similar to experimental
```

Comment:
- https://github.com/VirtusLab/scala-cli/issues/3258

Reply:
Let's add a link to the issue here.

Reply:
done